source/release-notes.txt

```diff
@@ -20,6 +20,28 @@ Release Notes for |bi|
 .. expect and we don't break the links on the downloads page.
 .. (DOCS-9670 for context)
 
+.. _bi-2-14-2:
+
+|bi| 2.14.2
+-----------
+
+*Released January 27, 2021*
+
+New Features
+~~~~~~~~~~~~
+
+- Adds the ``--configExpand`` command line option that enables the 
+  usage of expansion directives.
+
+- Adds support for ``__rest`` and ``__exec`` in 
+  `YAML <http://yaml.org/>`_ configurations.
+
```

Comment:
```suggestion
```
Let's leave these out of the changelog until the entire BI-2149 epic is complete